perllib/Open311/PopulateServiceList.pm

```diff
@@ -295,6 +295,23 @@ sub _set_contact_group {
     }
 }
 
+sub _groups_different {
+    my ($self, $old, $new) = @_;
+
+    my $diff = 1;
+    if ($old && $new) {
```

Comment:
Both old and new at this point could be a string or an arrayref. The comparison won't work if e.g. the current metadata is ["Streets"] and the fetched data is "Streets". Perhaps easier to make sure old/new are both arrayrefs and then compare?

Reply:
I did this.